### PR TITLE
Feature/raw prototype

### PR DESCRIPTION
**Pull request by jesrav**
Raw prototype of TUI (Text User Interface) on top of an MLFlow Tracking Server.

### CODE REVIEW

File: traktorml.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+from pydantic import BaseModel
+from textual.containers import ScrollableContainer
+from textual.events import Key
+from textual.widgets import DataTable, Footer
+from textual import on
+from textual.app import App, ComposeResult
+from textual.widgets import Header, Select
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
```
Comment (KPLauritzen) [inline traktorml.py]: You don't need both. `load_dotenv()` is enough

File: traktorml.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                is_reverse = False
+                run_table.sort(col_key, reverse=is_reverse)
+            else:
+                is_reverse = not self.experiment_view.current_sort[1]
+                run_table.sort(col_key, reverse=is_reverse)
+
+            self.experiment_view.current_sort = (col_key, is_reverse)
+            run_table.cursor_coordinate = cursor_coordinate
+
+        if event.key == "c":
```
Comment (KPLauritzen) [inline traktorml.py:145]: seems to be a lot of **almost** repetition in these `if event.key == ...` blocks. Can it be generalized.

File: traktorml.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+        ("r", "remove_row", "Remove Row"),
+        ("q", "quit", "Quit"),
+    ]
+
+    experiments = get_experiments()
+    selected_experiment: Union[Experiment, None] = None
+    experiment_views: Dict[str, ExperimentRunsView] = {}
+
+    @property
+    def experiment_view(self):
```
Comment (KPLauritzen) [inline traktorml.py]: Typehints for return value?

**Review by KPLauritzen — APPROVED** [review]

File: traktorml.py (hunk 1 above)
Comment (jesravn) [reply traktorml.py]: Good point.

File: traktorml.py (hunk 2 above)
Comment (jesravn) [reply traktorml.py:145]: You’re right, there is still a lot of repetition, even if I moved some of it into the set_data_table_to_experiment_view function. I think we should wait with more generalization until we experiment a bit more with textual.

File: traktorml.py (hunk 3 above)
Comment (jesravn) [reply traktorml.py]: I’ll add it:-)

**Pull request merged**